docs/en/v0.3/user-guide/monitoring.md

```diff
@@ -0,0 +1,136 @@
+# Metrics
+
+To collect GreptimeDB metrics using Prometheus
+
+## Binary
+
+Download the GreptimeDB binary, you can use the following command:
+```
+curl -fsSL \
+  https://raw.githubusercontent.com/greptimeteam/greptimedb/develop/scripts/install.sh | sh
+```
+
+Run GreptimeDB:
+```
+./greptime standalone start
+```
```

Comment:
remove the content above and add a link to getting-started/try-out-greptimeDB/binary